platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -40,6 +40,19 @@ The previous step will help you identify potential problems by issuing deprecati
 
 === Potential breaking changes
 
+==== Conventions removal
+
+The "convention" concept defined by the `org.gradle.api.plugins.Convention` type has been deprecated since Gradle 8.2 and is now removed.
```

Comment:
```suggestion
The "convention" concept, defined by the `org.gradle.api.plugins.Convention` type, has been deprecated since Gradle 8.2 and is now removed in Gradle 9.0.
```